sebsync.py

```diff
@@ -208,19 +249,40 @@ def sortable_author(author: str) -> str:
 def books_are_different(local_ebook: LocalEbook, remote_ebook: RemoteEbook) -> bool:
     """Return if differences are detected between local and remote ebooks."""
 
+    response = request(method="HEAD", url=remote_ebook.href)
+    remote_etag = response.headers.get("ETag", None)
+    time.sleep(
```

Comment:
Sending a series of `HEAD` requests means to me that we should probably find a better way to track changes.

Reply:
Yeah, that was the one part of using ETags that I wasn't keen on; good argument for dropping ETags.